published/npm.md

```diff
@@ -104,70 +105,70 @@ process](https://docs.npmjs.com/policies/disputes#beginning-the-process).
 In npm, the
 [package.json](https://docs.npmjs.com/cli/v6/configuring-npm/package-json)
 describes a project (name, components, dependencies, etc.). Dependencies may be
-defined by a package name, url, repo, etc. Additional constraints may be
-defined for each dependency, such as which tags, branches, versions or
-commitish are allowed.
+defined by a package name, URL, repo, etc. Additional constraints may be
+defined for each dependency, such as which tags, branches, versions, or
+commit-ish are allowed.
 
 **Note**: The manifest file ***does not*** list transitive dependencies; it
-lists only direct project dependencies.
+only lists direct project dependencies.
 
 ### Project types
 
 In the rest of this document, we will refer to three types of projects:
 
 - **Libraries**: These are projects published on the npm registry and consumed
-by other projects in the form of API calls. (Their manifest file 
-typically contains a `main`, `exports`, `browser`, `module`, and/or `types` entry).
+  by other projects in the form of API calls. (Their manifest file
+  typically contains a `main`, `exports`, `browser`, `module`, and/or `types` entry).
 
 - **Standalone CLIs**: These are projects published on the npm registry
-and consumed by end-users in the form of locally installed programs that
-are **always** run stand alone via `npx` or via a global install.
-An example would be [clipboard-cli](https://github.com/sindresorhus/clipboard-cli).
-(Their manifest file contains a `bin` entry).
+  and consumed by end-users in the form of locally installed programs that
+  are **always** run stand-alone via `npx` or via a global install.
+  An example would be [clipboard-cli](https://github.com/sindresorhus/clipboard-cli).
+  (Their manifest file contains a `bin` entry).
 
 - **Application projects**: These are projects that teams collaborate on in
-development and deploy, such as web sites and/or web applications. 
-An example would be a React web app for a company's user-facing SaaS.
-(Their manifest file typically contains `"private": true`).
+  development and deployment, such as websites and/or web applications.
+  An example would be a React web app for a company's user-facing SaaS.
+  (Their manifest file typically contains `"private": true`).
 
 ### Reproducible installation
 
-A reproducible installation is one that guarantees the exact same copy the
+A reproducible installation guarantees an exact, identical copy the
 dependencies are used each time the package is installed. This has various
 benefits, including:
 
 - Ensuring the dependencies installed are the ones declared and reviewed via
   pull requests.
 
-- Helping quickly identify possible compromises of your infrastructure if one
+- Helping quickly indentify possible compromises of your infrastructure if one
```

Comment:
@laurentsimon I think this was mistakenly added? or at least I'm not aware of the English meaning of `indentify`. Should we PR to fix it?

Reply:
good catch. Want to send the PR? tag me and I'll LGTM

Reply:
Sure thing: https://github.com/ossf/package-manager-best-practices/pull/38